src/dotnet/APIView/APIViewUnitTests/AppTests.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using ApiView;
+using Microsoft.TeamFoundation.Build.WebApi;
+using Xunit;
+
+namespace APIViewUnitTests
+{
+    public class AppTests
+    {
+        private string InputPath => Path.Combine("SampleTestFiles", "APIView.dll");
```

Comment:
What is this APIVIew.dll? Why do we need to check in this binary?

Reply:
Can't we directly run using local project path?

Reply:
The `ApiView.dll` is just a dll I'm using for testing (we could use any binary). I added a test where we use this dll as an input to the console app and check if it produces the result we expect.

Reply:
I think the point is that it would be better to have a human-readable artifact like a csharp class rather than the dll. The test could handle compiling the class using the CompilationFactory.

Reply:
@kinelski Can you please make the change as Josh suggested for test file instead of dll?

Reply:
Done. I'm consuming the asset files in the `ExactFormatting` folder now since they already cover multiple scenarios.